packages/edit-site/src/components/global-styles/font-library-modal/resolvers.js

```diff
@@ -7,13 +7,54 @@
  */
 import apiFetch from '@wordpress/api-fetch';
 
-export async function fetchInstallFont( data ) {
+export async function fetchInstallFontFamily( data ) {
 	const config = {
 		path: '/wp/v2/font-families',
 		method: 'POST',
 		body: data,
 	};
-	return apiFetch( config );
+	return apiFetch( config ).then( ( response ) => {
+		return {
+			id: response.id,
+			...response.font_face_settings,
+			fontFace: [],
+		};
+	} );
+}
+
+export async function fetchInstallFontFace( fontFamilyId, data ) {
+	const config = {
+		path: `/wp/v2/font-families/${ fontFamilyId }/font-faces`,
+		method: 'POST',
+		body: data,
+	};
+	return apiFetch( config ).then( ( response ) => {
+		return {
+			id: response.id,
+			...response.font_face_settings,
+		};
+	} );
```

Comment:
```suggestion
	const  response = await apiFetch( config );
	return {
		id: response.id,
		...response.font_face_settings,
	};
	};
```

See above. Should this be wrapped in a try / catch?

Reply:
Addressed the async/await a separate PR here: https://github.com/WordPress/gutenberg/pull/57961/commits/1d01919c0563e94f490c8e0d053b6db4f763f113.

I don't think these functions need to be wrapped in a try/catch block. Currently, in each place where these functions are invoked ([example here](https://github.com/WordPress/gutenberg/pull/57688/files#diff-ee8604b8acd83e45973351b79dc6983dc4daad84903a30a41e105a3d7429b74bR212)), they are wrapped in a try/catch block, where we're able to return specific and helpful error messages. If we wrapped them at a higher level, we would only be able to return more generic error messages.